setup.py

```diff
@@ -57,8 +57,8 @@
         "cachetools",
         "click",
         "dataclasses>=0.6;python_version<'3.7'",
-        "datacube>=1.8",
-        "eodatasets3 >= 0.17.0",
+        "datacube",
+        "eodatasets3",
```

Comment:
These are still valid, aren't they? They're essential minimums

Reply:
They'll still be relevant in the setup.py for regular pip installations (... and for when it goes to pypi)

Reply:
Hmm, I guess so. I get confused as to when to include in setup.py vs requirements.txt vs constraints.txt still! I'll put them back.